explainer.md

```diff
@@ -578,10 +623,12 @@ interface VRDevice : EventTarget {
 
 dictionary VRSessionCreateParametersInit {
   boolean exclusive = true;
+  HTMLCanvasElement outputCanvas = null;
```

Comment:
Primarily because the intent is to create a link to a specific canvas on the page, but I'm flexible on this point. Probably want to address @RafaelCintron's comments about the `ImageBitmapRenderingContext` before we make a decision here, though.